webapp/public/js/domjudge.js

```diff
@@ -674,3 +674,39 @@ function pinScoreheader()
 		}
 	}
 }
+
+function removeMultipleUpdater(self)
+{
+    /*var newUrl = '/jury/problems/';
+    var selectedNodes = $('.deleteSelector');
+    selectedNodes.each(function(index,element){
+        if (element.checked) {
+            newUrl += element.value+',';
+        }
+    });
+    newUrl = newUrl.slice(0,-1)+'/delete';
+    if(selectedNodes.length>0){
+        window.location = newUrl;
+    }*/
+    var oldUrl = $('#deleteMultiple').attr('href');
+    var postFix = '/delete'
+    var newUrl = ''
+    // We need to either append ,\d or remove ,\d
+    if(self.checked) {
+        if (oldUrl==''){
+            oldUrl = '/jury/problems//delete';
```

Comment:
nit: double slash

Also below

Reply:
In this case I make use of that, basicly it becomes:
'/jury/problems/'+problemId+'/delete'

In the final code I'll just make sure the if statement is moved in such a way that it doesn't bother you anymore.